core/templates/pages/topics-and-skills-dashboard-page/create-new-skill-modal.controller.spec.ts

```diff
@@ -112,6 +112,15 @@ describe('Create new skill modal', function() {
       expect($scope.errorMsg).toEqual(errorString);
     });
 
+  it('should return if the skill description is valid', function() {
+    $scope.newSkillDescription = 'valid';
+    expect($scope.isSkillDescriptionValid()).toBe(true);
+    $scope.newSkillDescription = 'invalid{{}}';
+    expect($scope.isSkillDescriptionValid()).toBe(false);
```

Comment:
Don't you have to assert the error message also? And maybe you can add stronger checks here, like spaces (as separate assertions)

Reply:
For the errors messages, there's a seperate test already present above. Also spaces are valid here.